PHPCI/Migrations/20150131075425_archive_project.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class ArchiveProject extends AbstractMigration
+{
+    /**
+     * Migrate Up.
+     */
+    public function up()
+    {
+        $project = $this->table('project');
+        $project->addColumn('archived', 'boolean');
```

Comment:
Firstly I've added "archive", than search in popular systems how to call column and decided to call it archive, but ok, I rename it to archived.